detection_rules/ecs.py

```diff
@@ -246,10 +251,14 @@ def get_event_type_hint(self, event_type, path):
 
 
 @cached
-def get_kql_schema(version=None, indexes=None, beat_schema=None) -> dict:
+def get_kql_schema(
+    version: str | None = None,
+    indexes: list[str] | None = None,
+    beat_schema: dict[str, Any] | None = None,
+) -> dict[str, Any]:
     """Get schema for KQL."""
-    indexes = indexes or ()
-    converted = flatten_multi_fields(get_schema(version, name='ecs_flat'))
+    indexes = indexes or []
```

Comment:
Curious as to why this was a tuple

Reply:
no idea. I don't think there is a risk of mutation, so we might as well simplify and have `list` here